tests/pkg_lvgl/main.c

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright (C) 2019 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       LittlevGL example application
+ *
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ *
+ * @}
+ */
+
+#include <string.h>
+
+#include "xtimer.h"
+
+#include "lvgl/lvgl.h"
+#include "lvgl_riot.h"
+
+#include "ili9341.h"
+#include "ili9341_params.h"
+#include "disp_dev.h"
+#include "ili9341_disp_dev.h"
+
+static ili9341_t dev;
+
+#define CPU_LABEL_COLOR     "FF0000"
+#define MEM_LABEL_COLOR     "0000FF"
+#define CHART_POINT_NUM     100
+
+/* Must be lower than LVGL_INACTIVITY_PERIOD_MS for autorefresh */
+#define REFR_TIME           200
+
+static lv_obj_t *win;
+static lv_obj_t *chart;
+static lv_chart_series_t * cpu_ser;
+static lv_chart_series_t *mem_ser;
+static lv_obj_t *info_label;
+static lv_task_t *refr_task;
+
+#define SYSMON_STRING_BUFFER_SIZE 256
+
+static void sysmon_task(lv_task_t *param)
+{
+    (void)param;
+
+    /* Force a wakeup of lvgl when each task is called: this ensure an activity
+       is triggered and wakes up lvgl during the next LVGL_ACTIVITY_PERIOD ms */
+    lvgl_wakeup();
```

Comment:
shouldn't lvgl be woken up *after* queueing changes?

Reply:
I moved it after.